process-engine/process_engine_runtime
# Add runtime setup

## What did you change?

* added a setup similar to the skeleton setup
* modified the setup to use sqlite databases
* modified the setup to use the production environment
* modified the setup to store database files in the same userData folder as electron
   * on OS X: `~/Library/Application Support/process_engine_runtime`

## How can others test the changes?

* clone the package
* run `npm install -g @process-engine/process_engine_runtime@feature~add_runtime_setup`
* run `process-engine`
   * you should see log output containing
       `info: [bootstrapper] Bootstrapper started successfully.`

## PR-Checklist

Please check the boxes in this list after submitting your PR:
